services/webnn/ort/error_ort.h

```diff
@@ -24,22 +24,33 @@ namespace webnn::ort {
     }                                                                \
   } while (0);
 
-#define ORT_CALL_FAILED(ort_func)                                  \
-  [&]() -> ort::ScopedOrtStatusPtr {                               \
-    ort::ScopedOrtStatusPtr status(ort_func);                      \
+#define LOG_IF_FAILED(ort_call)                                    \
+  do {                                                             \
+    ort::ScopedOrtStatusPtr status(ort_call);                      \
     if (status) {                                                  \
-      LOG(ERROR) << "[WebNN] Failed to call " << #ort_func << ": " \
+      LOG(ERROR) << "[WebNN] Failed to call " << #ort_call << ": " \
+                 << ort::GetOrtApi()->GetErrorMessage(status);     \
+    }                                                              \
```

Comment:
You may still want to return the status, so other macros can reuse the log-if-failed code. If you think the current name doesn't reflect it returns the status, you probably can just name it as `CALL_ORT_FUNC`.